dydx/dydxStateManager/dydxStateManager/AbacusState+Combine.swift

```diff
@@ -142,24 +143,45 @@ public final class AbacusState {
      Subaccount
      **/
     public func subaccount(of subaccountNumber: String) -> AnyPublisher<Subaccount?, Never> {
-        account
-            .compactMap(\.?.subaccounts?[subaccountNumber])
-            .removeDuplicates()
-            .share()
-            .eraseToAnyPublisher()
+        if dydxBoolFeatureFlag.enable_isolated_margins.isEnabled {
+            account
```

Comment:
@mike-dydx  FYI, Abacus has a function that returns the subaccount:  `PerpetualState.subaccount()`.